### DISCUSSION TITLE
Searching for multiple names using "Org"

### DISCUSSION THREAD

**Discussion by kmichaelpratt** [shadowscatcher/shodan#15] [General]
Hello, thanks for the hard work with this. Question for you, I trying to do a search using multiple Org names under one query for example here is a GET request that works 

https://api.shodan.io/shodan/host/search?key=APIKEY1&query=org:'Digital Ocean','Equinix'&facets=ip

So far it appears that when I attempt to utilize the same type query, under the query Struct via the Search package with Org, I am only able to search one string IE "'Digital Ocean" or "Equinix" but not both in the same query? Is that correct? Thanks for any help, this is very well put together.

**Comment by kmichaelpratt** [comment]
@shadowscatcher Please see above when you have time.

**Reply by shadowscatcher** [reply]
Hi, @kmichaelpratt , thanks for bringing this up. Yes, if you just set Org field twice, the value will be overriden by the latter. You can work around that by putting a comma between these values. Or, even better, use `Text` field instead and set raw query like `"org:Digital Ocean,Equinix"` if you want to put logical OR between values or `"org:first org:second"` for logical AND (I doubt that banners can belong to two orgs in Shodan, however).

I recommend using `Text` field every time you can't simply set query fields or want to use new parameter that is not yet supported. For instance, when you want to exclude something from results.

See these links for advanced search docs:
https://www.shodan.io/search/filters
https://www.shodan.io/search/advanced
